docs/testing/troubleshoot-manifest.md

```diff
@@ -63,6 +63,35 @@ You can use runtime logging to debug your add-in's manifest as well as several i
 > [!IMPORTANT]
 > Runtime Logging affects performance. Turn it on only when you need to debug issues with your add-in manifest.
 
+### Use runtime logging from the command line
+
+Enabling runtime logging from the command line is the fastest way to use this logging tool. The below commands use npx, which is provided by default as part of npm@5.2.0+. If you have an earlier version of [npm](https://www.npmjs.com/), try [Runtime logging on Windows](#runtime-logging-on-windows) or [Runtime logging on Mac](#runtime-logging-on-mac) instructions, or [install npx](https://www.npmjs.com/package/npx).
```

Comment:
Second sentence: change "The below commands" to "The following commands".

Reply:
Third sentence:
- Consider moving the part about installing npx to a new sentence preceding this information. That way, it reads: this uses npx, here's how to install npx, and can also do these other things.
- Consider changing "try" to "follow the advice in".